models/demos/mamba/tests/test_mamba_demo.py

```diff
@@ -20,3 +20,27 @@ def test_demo(user_input, model_version, device, use_program_cache, get_tt_cache
         display=False,
         cache_dir=get_tt_cache_path(model_version),
     )
+
+
+@pytest.mark.skip("Prefill Decode Demo is not supported for CI")
```

Comment:
Can we not enable this?

Reply:
We can enable it actually. Just need to turn off display

Reply:
Ok lets enable it, more tests the better!